cuda/solver/common_trs_kernels.cuh

```diff
@@ -143,6 +149,17 @@ struct CudaSolveStruct : gko::solver::SolveStruct {
                matrix::Dense<ValueType>* output, matrix::Dense<ValueType>*,
                matrix::Dense<ValueType>*) const
     {
+        if (input->get_size()[1] != num_rhs) {
+            throw gko::ValueMismatch{
```

Comment:
In effect, this now means there is a `num_rhs` factory parameter requirement to be specified with triangular solvers (or CUDA will throw). Should I then list this change in the "Deprecations and important changes" of the release?

Reply:
We should probably mention it, yes

Reply:
I tried to mention this change in https://github.com/ginkgo-project/ginkgo/discussions/1177

Reply:
I extended the explanation a bit, thanks